### PR TITLE
[OpenVINO-EP] Update version to OpenVINO 2019 R2

### PR DESCRIPTION
**Pull request by psfoley**
**Description**: Added support for OpenVINO 2019 R2

**Motivation and Context**
- This PR brings the OpenVINO Execution Provider up to date with the latest version of OpenVINO. It also maintains support for legacy versions OpenVINO 2019 R1.1 and 2018 R5. 
- Several bug fixes and security fixes
- The latest OpenVINO 2019 R2 Release Notes can be found here: [https://software.intel.com/en-us/articles/OpenVINO-RelNotes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes)

### CODE REVIEW

File: onnxruntime/core/providers/openvino/openvino_execution_provider.cc [hunk 1]
```diff
@@ -292,6 +293,15 @@ void CheckGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::stri
       }
     }
 
+    //Add and Mat is only supported if it is followed by Add
```
Comment (TMVector) [inline onnxruntime/core/providers/openvino/openvino_execution_provider.cc]:
```suggestion
    //Add and Mul don't support int64 inputs
```

File: onnxruntime/core/providers/openvino/openvino_execution_provider.cc [hunk 2]
```diff
@@ -322,13 +332,13 @@ void CheckGraphSupported(const onnxruntime::GraphViewer& graph_viewer, std::stri
     }
 
     //Dropout , Identity and Concat can't have graph inputs
```
Comment (TMVector) [inline onnxruntime/core/providers/openvino/openvino_execution_provider.cc]:
```suggestion
    //Dropout, Identity, Concat, and Gemm can't have graph inputs
```

File: onnxruntime/core/providers/openvino/openvino_mo/openvino_mo.py [hunk 3]
```diff
@@ -377,6 +406,106 @@ def driver_R1(onnx_modelproto_bytes, precision: str, output_model_name: str, out
...
+    try:
+        model_proto = onnx.load_from_string(bytes(onnx_modelproto_bytes))
+    except Exception as e:
+        print("[python] onnx exception: ", str(e))
+
+    model_graph = model_proto.graph  # pylint: disable=no-member
+    log.debug("Number of nodes in graph_def: {}".format(len(model_graph.node)))
+    log.debug("Number of all input ports (not true inputs) in graph_def: {}".format(len(model_graph.input)))
+    log.debug("Number of initializers in graph_def: {}".format(len(model_graph.initializer)))
+    log.debug("Number of real inputs in graph_def: {}".format(len(model_graph.input) - len(model_graph.initializer)))
```
Comment (TMVector) [inline onnxruntime/core/providers/openvino/openvino_mo/openvino_mo.py]: Note this [doesn't hold](https://github.com/onnx/onnx/commit/dc75285d4a1cff9618400164dfdb26c5a1bab70a) for IR version >= 4

**Comment by jywu-msft** [comment]
please resolve conflict in openvino_graph.cc

**Pull request closed without merging**